### PR TITLE
start working on project management guides

### PR DESCRIPTION
**Pull request by benigeri**

### CODE REVIEW

File: project-management.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# SVI's Project Management Workflow
+
+This is how we interface with clients, run sprints, and track the progress
+of our projects.
+
+This guide is just a start, and it's meant to change as we grow and learn. **If you disagree with anything, open an issue.** In the meantime, stick to the guide. Please see the [contributing guidelines](/contributing.md) for more info.
+
+## Sprint Timeline
+
+### Pre-Sprint
```
Comment (JohnYangSam) [inline project-management.md]: We'll need to make this more clear. This is okay at a first pass.

@piafaustino please hop in to help us work on this. I'm counting on your to help take your fresh perspective, clean this up, and help us make a better process and clearer prose for newly onboarded members.

File: project-management.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+- The PM of the project should have organized:
```
Comment (JohnYangSam) [inline project-management.md]: We should do a checklist at each of these points:

For what should be in emails, what should be covered at each sprint, etc.

We should also have templates for emails, meeting notes, and agendas for meetings.

@piafaustino

File: project-management.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+### Day 0: First day of sprint
+- All members of the engineering team get together on Skype facilitated by the PM
+- PM and tech lead facilitate a the list of issues and assignment over the two weeks among engineering
+- The PM should add all of the issues that need to be addressed during the sprint to a sprint milestone
+- **Deliverable:** Email update to the client with finalized sprint details
+
+### Day 7: Mid-sprint checkin
+- All members of the engineering team get together on Skype facilitated by the PM
+- Everybody answers:
+ - What are you blocked on?
```
Comment (JohnYangSam) [inline project-management.md]: Need checklist.

File: project-management.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+- The PM of the project should have organized:
+ - A long term list of target components, issues, and dates with the client
+ - A short term list of target components and issues with the client to aim to be completed at the end of the sprint
+- The PM is responsible for making sure that the short term and long term goals are driven by the feasibility assessed by engineering and balance the expecations of the client appropriately
+
+### Day 0: First day of sprint
+- All members of the engineering team get together on Skype facilitated by the PM
+- PM and tech lead facilitate a the list of issues and assignment over the two weeks among engineering
+- The PM should add all of the issues that need to be addressed during the sprint to a sprint milestone
+- **Deliverable:** Email update to the client with finalized sprint details
```
Comment (JohnYangSam) [inline project-management.md]: @benigeri - we should have a way to highlight the deliverable for each week and delinate internal steps and steps that need to go to the client.

File: project-management.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+- Features should be described as a user story.
+- Break large features into several smaller ones, and keep them as self-contained as possible. Each feature should have the granularity of 1 day max— if it doesn't break it down further.
+- The description should include all of the information needed to implement the feature.
+
+#### `type: other`
+- Use the other label for UI improvements, performance improvements, refactoring, security improvements, or anything else.
+- We can always add new types if `type: other` becomes overused.
+
+#### Add a priority label
+- `priority: HIGH`: Critical, needs to be addressed immediately.
```
Comment (JohnYangSam) [inline project-management.md]: Do we prefer this or 1st, 2nd, low?

**Comment by JohnYangSam** [comment]
@anjgarong @piafaustino @stefsy please do hop in and help us form this.

The current guide is very rough and I am sure you have lots of good input to improve and change this around scrum and the FGI project, so feel free to add to this pull request or open a new one.

File: project-management.md (hunk 5 above)
Comment (benigeri) [inline project-management.md]: I personally prefer this because to me 2nd priority still seems really high, but I don't feel strongly.

**Comment by piafaustino** [comment]
Ack-- just saw all these mentions now for some reason.Sorry for the late jump in! Noted. :)

**Pull request merged**